packages/aws-cdk-lib/aws-eks/README.md

```diff
@@ -206,6 +206,27 @@ cluster.addNodegroupCapacity('custom-node-group', {
 });
 ```
 
+To define the maximum number of instances which can be simultaneously replaced in a node group during a version update you can set `maxUnavailable` or `maxUnavailablePercentage` options.
+
+> For more details visit [Nodegroup UpdateConfig](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-eks-nodegroup-updateconfig.html)
```

Comment:
```suggestion
> For more details visit [Updating a managed node group](https://docs.aws.amazon.com/eks/latest/userguide/update-managed-node-group.html)
```
I think it should point to the user guide since CloudFormation links are already provided on the properties definition.